etc/config/circt.amazon.properties

```diff
@@ -0,0 +1,13 @@
+compilers=&circtopt
+defaultCompiler=circtopt-trunk-20220713
+compilerType=mlir
+
+supportsBinary=false
+supportsExecute=false
+supportsAsmDocs=false
+
+group.circtopt.compilers=circtopt-trunk-20220713
+group.circtopt.isSemVer=false
+group.circtopt.baseName=CIRCT opt
+
+compiler.circtopt-trunk-20220713.exe=/opt/compiler-explorer/circt-circt-trunk-20220713/bin/circt-opt
```

Comment:
I suspect this is just going to be /opt/compiler-explorer/circt-trunk/ and it will be refreshed every day

Reply:
so it could also just be called circtopt-trunk

Reply:
I pushed up a commit to rename that and change the path to `/opt/compiler-explorer/circt-trunk/`. I wasn't sure exactly what the path should be. Is that something I can find in compiler-explorer/infra CI logs or something?